syntax = "proto3";

package proto;

service Pay {
    rpc AddPayment(NewPayment) returns (PayResponse);
}

message NewPayment {
    string username = 1;
    string idVolo = 2;
    repeated string selectedSeats = 3;
    string paymentDate = 4;
    string basePrice = 5;
    string seatsPrice = 6;
    string servicesPrice = 7;
    string totalPrice = 8;
    int32 numStivaMedi = 9;
    int32 numStivaGrandi = 10;
    int32 numBagagliSpeciali = 11;
    int32 numAssicurazioni = 12;
    int32 numAnimali = 13;
    int32 numNeonati = 14;
    string email = 15;
}

message PayResponse {
    bool isOk = 1;
}
